crates/re_data_store/src/store_read.rs

```diff
@@ -454,7 +454,9 @@ impl IndexedTable {
     pub fn find_bucket(&self, time: TimeInt) -> (TimeInt, &IndexedBucket) {
         // This cannot fail, `iter_bucket` is guaranteed to always yield at least one bucket,
         // since indexed tables always spawn with a default bucket that covers [-∞;+∞].
-        self.range_buckets_rev(..=time).next().unwrap()
+        self.range_buckets_rev(..=time)
+            .next()
+            .expect("this cannot fail")
```

Comment:
Better to use `#[allow(clippy::unwrap_used)]` for things like these imho (slightly less code generated)